### PR TITLE
Some last minute fixes

### PR DESCRIPTION
**Pull request by nnunn**
adding ejournals and databases into main search (for the bento removal). Fixing library codes in locations.yml

### CODE REVIEW

File: config/locations.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ covenantmh: Covenant Health - Misericordia Community Hospital
 cross:  Cross Cancer Institute
 cuc:  Canadian University College
 gprc: Grande Prairie Regional College
+gprcgp: Grande Prairie Regional College
```
Comment (weiweishi) [inline config/locations.yml]: Is this intended? to have two codes for Grande Prairie Regional College?

**Comment by weiweishi** [comment]
sounds good.


*Weiwei Shi*Digital Initiative Applications Librarian

2-10L Cameron Library, University of Alberta
780-492-7802  |  weiwei.shi@ualberta.ca
*"The University of Alberta respectfully acknowledges that we are situated
on Treaty 6 territory, traditional lands of First Nations and Métis
people."*

File: config/locations.yml (hunk 1 above)
Comment (nnunn) [reply config/locations.yml]: Sadly this isn't the most recent list Sean -- there are codes that don't appear in that list that are coming up on search. For example -- ahserc doesn't appear on the list but does appear in the results. I have added a listing for it as Alberta Health Services, but this is a duplication... thoughts?

**Comment by seanluyk** [comment]
If Elaine is around it might be worth asking about specific codes. She sent
the list to me just this week.

Sean Luyk

**Comment by nnunn** [comment]
Ok that last commit fixes all the library codes *but* there are duplications of some libraries. Especially Alberta Health Services (I did remove the Grande Prairie duplication as I think they have just split their location into three different locations). I also had to add a really strange location which is obviously just a mistake - usic tech lab: University of Alberta Cameron - Science & Technology - Lab. This at least gets rid of the awful looking symphony codes in the library facet, but I will check with Elaine about the missing codes. I think though we should still get this version into test right away.

**Comment by seanluyk** [comment]
Agreed and we'll need to work out a process for keeping these up to date.
Thanks Natasha!

Sean Luyk

**Pull request merged**